docs/conf/portland/2024/schedule.rst

```diff
@@ -4,4 +4,257 @@
 Schedule
 ========
 
-{% include "conf/schedule.rst" %}
+Write the Docs is more than a conference.
```

Comment:
I don't know why we're moving things back into `portland/*` ? If it's an updating problem we could include /sechedule2024.rst` or similar?